docs/markdown/home.md

```diff
@@ -1,6 +1,6 @@
 ### AngularJS-Portal Documentation
 
-Thanks for visiting the documentation for [angularjs-portal](https://github.com/UW-Madison-DoIT/angularjs-portal). This documentation describes the features and implementation details of the angularjs-portal project, the new home of MyUW. angularjs-portal is based on uw-frame, if you want to learn about that I suggest looking at [that docs page](madison-doit.github.io/uw-frame/).
+Thanks for visiting the documentation for [angularjs-portal](https://github.com/UW-Madison-DoIT/angularjs-portal). This documentation describes the features and implementation details of the angularjs-portal project, the new home of MyUW. angularjs-portal is based on uw-frame, if you want to learn about that I suggest looking at [that docs page][uw-frame docs].
```

Comment:
That isn't a valid URL either... should it be `http://uw-madison-doit.github.io/uw-frame/`